### ISSUE TITLE
Test cloud compute node (static server)

### ISSUE THREAD

**Issue by ElliotAOram** [ISISScientificComputing/autoreduce#326]
Issue raised by: [developer]

### What?
The compute node needs to be tested in a cloud environment. To start with we should just use a static server in the cloud. If we don't have an ansible script for only compute node, we need to make a new issue on github and make a separate ansible script just for the compute node (this should be done before continuing with this issue).

### Where?
Ansible script

### How to test the issue is resolved
1. Stop the Queue processor on the actual development node
1. Set up a cloud instance
1. Install the Compute node only 
1. Redirect the end of run monitor and webapp nodes for the development environment to look at the cloud compute node.
1. Manually submit runs and ensure they arrive in the database / webapp
1. Wait for the End of Run monitor to see if any runs are picked up and reduced by the cloud node

**Comment by dtasev** [comment]
Done via #1022 and kind of a dupe of #327
